internal/xds/xdsclient/pool.go

```diff
@@ -61,10 +61,6 @@ type OptionsForTesting struct {
 	// Name is a unique name for this xDS client.
 	Name string
 
-	// WatchExpiryTimeout is the timeout for xDS resource watch expiry. If
-	// unspecified, uses the default value used in non-test code.
-	WatchExpiryTimeout time.Duration
```

Comment:
I'd rather that we not delete this field and instead use this to set the value in the generic xDS client (either directly or through a setter provided by the latter).